src/charts/trends/trends-chart.directive.js

```diff
@@ -114,7 +139,34 @@ angular.module('patternfly.charts').directive('pfTrendsChart',
         showYAxis: '=?'
       },
       replace: true,
-      templateUrl: 'charts/trends/trends-chart.html'
+      templateUrl: 'charts/trends/trends-chart.html',
+      controller: ['$scope',
+        function ($scope) {
+          $scope.getPercentageValue = function () {
+            return Math.round($scope.getLatestValue() / $scope.chartData.total * 100.0);
+          };
+          $scope.getLatestValue = function () {
+            return $scope.chartData.yData[$scope.chartData.yData.length - 1];
+          };
+          $scope.getChartHeight = function () {
+            var retValue = 60;
```

Comment:
Hi, I tried using the 'const' keyword, but our PhantomJS/Karma Unit Test browers throws the following error:
```
PhantomJS 1.9.8 (Linux) ERROR
  SyntaxError: Parse error
  at /home/dtaylor/Dev/repos/angular-patternfly/src/charts/trends/trends-chart.directive.js:132

Warning: Task "karma:unit" failed. Use --force to continue.

Aborted due to warnings.
~/Dev/repos/angular-patternfly>node /home/dtaylor/Dev/repos/angular-patternfly/src/charts/trends/trends-chart.directive.js

/home/dtaylor/Dev/repos/angular-patternfly/src/charts/trends/trends-chart.directive.js:132
    const SMALL = 30, LARGE = 60;
    ^^^^^
SyntaxError: Use of const in strict mode.
    at Module._compile (module.js:439:25)
```
Which lead me to read about how '...brower support is heavily fractionary'.  So, I just used a 'var' until I can figure out how to implement 'const'.